xata/app/entry.client.tsx

```diff
@@ -0,0 +1,4 @@
+import { RemixBrowser } from '@remix-run/react'
+import { hydrateRoot } from 'react-dom/client'
+
+hydrateRoot(document, <RemixBrowser />)
```

Comment:
This is from React 18, while package.json is installing React 17

Reply:
Please revert to v17 again